### ISSUE TITLE
LSTM

### ISSUE THREAD

**Issue by txir-met** [MetOffice/ML-TC#7]
The LSTM code is based on [this tutorial](https://towardsdatascience.com/video-prediction-using-convlstm-with-pytorch-lightning-27b195fd21a2) and [this github repo](https://github.com/holmdk/Video-Prediction-using-PyTorch).

An attempt to bring the tutorial libraries up to date was not fruitful, and caused the network to train more quickly and significantly less well, with large white sections. The exact source of the issue was not identified, so the old library versions were used.

The network struggles with image sharpness, giving blurry predictions for both the MNIST dataset in the tutorial, and our own Cyclone data. In our own data, the output is usually a somewhat blurrier copy of the most recent input.

The performance does not drop significantly loss-wise when moving from one-hourly to three-hourly steps. The loss fell below 0.005, so it may be worth attempting to introduce a different/additional loss heuristic in the future.

Overall, this approach was not the most promising so was not investigated in much greater depth, and we stuck with 64x64 pixels.
